task-3/src/event-emitter/eventEmitter.ts

```diff
@@ -0,0 +1,73 @@
+type IEventName = string | symbol;
+type IEventFunction = (...args: any[]) => void;
+interface IListeners {
+  [key: IEventName]: IEventFunction[] | undefined;
+}
+
+class EventEmitter {
+  listeners: IListeners = {};
+
+  addListener(eventName: IEventName, fn: IEventFunction): EventEmitter {
+    let eventListeners = this.listeners[eventName];
```

Comment:
```suggestion
    const eventListeners = this.listeners[eventName];
```
The eventListeners variable is never modified in this file

Reply:
Fixed